ksql-engine/src/main/java/io/confluent/ksql/analyzer/QueryValidator.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2019 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.analyzer;
+
+import io.confluent.ksql.parser.tree.Query;
+import io.confluent.ksql.parser.tree.Sink;
+import java.util.Optional;
+
+/**
+ * Vaidator used by {@link QueryAnalyzer}.
```

Comment:
typo "Vaidator".

This comment doesn't have much useful info, probably not worth adding?